### PR TITLE
Dynamic menu creation

### PR DESCRIPTION
**Pull request by manishks12288**
@coolbung Sqltorage instead of localstorage, Pull to refresh with on/off config, Searchbar with on/off config, Toast for error handling implemented.

### CODE REVIEW

File: app/unite-framework/unitemenu.ts [hunk 1]
```diff
@@ -1,26 +1,48 @@
...
 import {Injectable, Inject} from 'angular2/core';
-import {Http} from 'angular2/http';
-import 'rxjs/add/operator/map';
-// import {GettingStartedPage} from '../pages/getting-started/getting-started';
-// import {ListPage} from '../pages/list/list';
-
-
 @Injectable()
 export class UniteMenu {
   pages: any;
```
Comment (coolbung) [inline app/unite-framework/unitemenu.ts]: Needs to be something like below. Add all needed fields.

pages: Array<{title: string, component: string, icon: string}>;

File: app/app.ts [hunk 2]
```diff
@@ -1,9 +1,10 @@
-import {App, IonicApp, Platform} from 'ionic-angular';
+import {App, IonicApp, Platform, LocalStorage, Storage, Events} from 'ionic-angular';
```
Comment (coolbung) [inline app/app.ts]: Use SqlStorage instead of LocalStorage everywhere

File: app/unite-framework/unitelist.ts [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
 import {Http} from 'angular2/http';
 import 'rxjs/add/operator/map';
 
 @Injectable()
 export class UniteList {
   selectedItem: any;
   icons: string[];
-  items: Array<{title: string, note: string, icon: string}>;
-  url: string;
+  items: Array<{ title: string, note: string, icon: string }>;
```
Comment (coolbung) [inline app/unite-framework/unitelist.ts]: @manishks12288 this is still not right. Need to add all supported variables for a menu. Missing parent, children, order, component and a few more

File: app/unite-framework/unitemenu.ts (hunk 1 above)
Comment (coolbung) [inline app/unite-framework/unitemenu.ts]: @manishks12288 this needs to be fixed. Need to add all supported variables for a menu. Missing parent, children, order, component and a few more

File: app/app.ts (hunk 2 above)
Comment (manishks12288) [inline app/app.ts]: @coolbung Use SqlStorage instead of LocalStorage=> (Done)

**Comment by manishks12288** [comment]
@coolbung updated README.md

File: app/unite-framework/unitemenu.ts (hunk 1 above)
Comment (manishks12288) [inline app/unite-framework/unitemenu.ts]: @coolbung Need to change pages object node dynamically. So i kept it same (pages: any)

**Pull request merged**